qt/aqt/editcurrent.py

```diff
@@ -33,9 +30,39 @@ def __init__(self, mw: aqt.AnkiQt) -> None:
         self.editor.card = self.mw.reviewer.card
         self.editor.set_note(self.mw.reviewer.card.note(), focusTo=0)
         restoreGeom(self, "editcurrent")
+        self.setupButtons()
         gui_hooks.operation_did_execute.append(self.on_operation_did_execute)
         self.show()
 
+    def setupButtons(self) -> None:
+        if self.editor.current_notetype_is_image_occlusion():
+            bb = self.form.buttonBox
+            ar = QDialogButtonBox.ButtonRole.ActionRole
+            # add io hide all button
+            self.ioAddButton = bb.addButton(tr.actions_update_note(), ar)
+            qconnect(self.ioAddButton.clicked, self.onAddIo)
+            self.ioAddButton.setShortcut(QKeySequence("Ctrl+Shift+I"))
```

Comment:
I'm confused by the different way this is handled in the editcurrent and browse dialogs. Why do we have an explicit update button in one, but update automatically in the other? Maybe updates could just be automatic in both?

Reply:
Now the add button only added to Add Cards, the other mode edit and browse mode does not have add button. The only downside for this is that if user want to change the `hide all` to `hide one` note then they have to delete and add again.
Or a more button in ImageOcclusionPage can be added to switch between `hide all` and `hide one` notetype.